content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+  weight: 10
+---
+
+<!-- overview -->
+
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
+कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
+काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेट्स वातावरण प्रदान करता है।
+
+{{< note >}}
+यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
+स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+* मिनीक्यूब में एक नमूना एप्लीकेशन डेप्लॉय करें।
+* ऐप को चलाएं।
+* एप्लिकेशन लॉग देखें।
+
+## {{% heading "prerequisites" %}}
+
+
+यह ट्यूटोरियल एक कंटेनर इमेज प्रदान करता है जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है।
+
+
+
+<!-- lessoncontent -->
+
+## एक मिनीक्यूब क्लस्टर बनाएं
+
+1. **Launch Terminal** पर क्लिक करें।
+
+    {{< kat-button >}}
+
+{{< note >}}
+यदि आपने स्थानीय रूप से मिनीक्यूब स्थापित किया है, तो `minikube start` चलाएँ। इससे पहले कि आप `minikube dashboard` चलाएं, आपको एक नया टर्मिनल खोलना चाहिए, वहां `minikube dashboard` शुरू करना चाहिए, और फिर मुख्य टर्मिनल पर वापस जाना चाहिए।
+{{< /note >}}
+
+2. ब्राउज़र में कुबेरनेट्स डैशबोर्ड खोलें:
+
+    ```shell
+    minikube dashboard
+    ```
+
+3. केवल काटाकोडा वातावरण के लिए: टर्मिनल फलक के शीर्ष पर, प्लस(+) चिह्न पर क्लिक करें, और फिर **Select port to view on Host 1** क्लिक करें।
+
+4. केवल काटाकोडा वातावरण के लिए: `30000` टाइप करें, और फिर **Display Port** क्लिक करें।
+
+{{< note >}}
+`dashboard` कमांड डैशबोर्ड ऐड-ऑन को इस्तेमाल के लिए तैयार करता है और प्रॉक्सी को डिफ़ॉल्ट वेब ब्राउज़र में खोलता है।
+आप डैशबोर्ड पर कुबेरनेट्स संसाधन जैसे डेप्लॉयमेंट और सर्विस बना सकते हैं।
+
+यदि आप रुट(root) के रूप में इस्तेमाल कर रहे हैं, तो [URL से डैशबोर्ड खोलना](#open-dashboard-with-url) देखें।
+
+आमतौर पर, डैशबोर्ड केवल आंतरिक कुबेरनेट्स वर्चुअल नेटवर्क के भीतर से ही पहुँचा जा सकता है।
+डैशबोर्ड को कुबेरनेट्स वर्चुअल नेटवर्क के बाहर से एक्सेस करने योग्य बनाने के लिए `dashboard` कमांड एक अस्थायी प्रॉक्सी बनाता है।
+
+प्रॉक्सी को रोकने और प्रक्रिया से बाहर निकलने के लिए `Ctrl+C` का प्रयोग करें।
+कमांड से बाहर निकलने के बाद, डैशबोर्ड कुबेरनेट्स क्लस्टर में चलता रहता है। 
+आप डैशबोर्ड तक पहुंचने के लिए एक और प्रॉक्सी बनाने के लिए फिर से `dashboard` कमांड चला सकते हैं।
+{{< /note >}}
+
+## URL से डैशबोर्ड खोलें {#open-dashboard-with-url}
+
+यदि आप वेब ब्राउज़र नहीं खोलना चाहते हैं, तो URL प्राप्त करने के लिए url फ़्लैग के साथ `dashboard` कमांड चलाएँ:
+
+```shell
+minikube dashboard --url
+```
+
+## डिप्लॉयमेंट बनाएँ
+
+कुबेरनेट्स [*पॉड*](/hi/docs/concepts/workloads/pods/) एक या अधिक कंटेनरों का एक समूह है, 
+जो प्रशासन और नेटवर्किंग के उद्देश्यों के लिए एक साथ बंधे होते हैं। इस ट्यूटोरियल के 
+पॉड में केवल एक कंटेनर है। कुबेरनेट्स 
+[*डिप्लॉयमेंट*](/hi/docs/concepts/workloads/controllers/deployment/) आपके पॉड के स्वास्थ्य की 
+जाँच करता है और यदि पॉड बंद हो जाता है तो पॉड के कंटेनर को पुनः आरंभ करता है।
+पॉड्स के निर्माण और स्केलिंग को प्रबंधित करने के लिए डिप्लॉयमेंट अनुशंसित तरीका है।
```

Comment:
```suggestion
पॉड्स के निर्माण और स्केलिंग को प्रबंधित करने के लिए डिप्लॉयमेंट अनुशंसित तरीका है।
```

My concern with the word डिप्लॉयमेंट is precisely this. We are using डेप्लॉय in some places and an alternate spelling in others. This needs to be standardized.

Reply:
+1 on standardization.

Reply:
Thanks for pointing, changing it to डेप्लॉयमेंट